File: portafolio-backend/src/main/java/com/portafolio/portafoliobackend/dtos/TblEducacionDTO.java
package com.portafolio.portafoliobackend.dtos;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class TblEducacionDTO {
    
    // CAMPO IDENTIFICADOR DE EDUCACION
    private Long idEducacion;

    // IDENTIFICADOR INTERNO DEL PERFIL
    private Long idPerfil;

    // DESCRIBIR TU EDUCACIÓN
    private String deEducacion;
}

File: portafolio-frontend/src/app/models/TblEducacionDTO.ts
export class TblEducacionDTO {

  // CAMPO IDENTIFICADOR DE EDUCACION
  idEducacion: number;

  // IDENTIFICADOR INTERNO DEL PERFIL
  idPerfil: number;

  // DESCRIBIR TU EDUCACIÓN
  deEducacion: string;

  // CAMPO ESTADO DEL REGISTRO. LOS POSIBLES VALORES SON: "1" = ACTIVO Y "0" =
  // INACTIVO
  esRegistro: string;

  // CAMPO USUARIO CREADOR
  usCreacion: string;

  // CAMPO TERMINAL DE CREACIÓN
  ipCreacion: string;

  // CAMPO FECHA Y HORA DE CREACIÓN
  feCreacion: Date;

  // CAMPO USUARIO MODIFICADOR
  usActualizacion: string;

  // CAMPO TERMINAL DE MODIFICACIÓN
  ipActualizacion: string;

  // CAMPO FECHA Y HORA DE MODIFICACIÓN
  feActualizacion: Date;
}
